src/screens/GroupInfo/GroupMedia.js

```diff
@@ -104,7 +104,7 @@ const styles = StyleSheet.create({
   },
 
   viewborderbottom: {
-    borderBottomColor: colors.holytosca,
+    borderBottomColor: COLORS.holyTosca,
     borderBottomWidth: 2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, replacing hard-coded color values with references to color constants from a theme file is a good practice. It enhances code readability and maintainability.
```diff
- borderBottomColor: colors.holytosca,
+ borderBottomColor: COLORS.holyTosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->